### ISSUE TITLE
Python Mode, No Display Window

### ISSUE THREAD

**Issue by wutiship** [jdf/Processing.py-Bugs#285]
Windows 8 (64 bit), Processing 3.5.4, Python Mode 3056.  
Display window doesn't show up, not even with simple test scripts. Everything works alright in java mode though. I tried to block wifi connection/firewall and reinstall processing with older stable versions.

**Comment by paparico** [comment]
i had the same (or a similar) problem on a linux (kubuntu) platform.
the sketch execution window was started "minimized", and wasnt restored when clicking its icon in the taskbar. 
i had to rightclick the icon and deselect the "minimized" option in the "more options" section.
hope it helps.

**Comment by wutiship** [comment]
> 
> 
> i had the same (or a similar) problem on a linux (kubuntu) platform.
> the sketch execution window was started "minimized", and wasnt restored when clicking its icon in the taskbar.
> i had to rightclick the icon and deselect the "minimized" option in the "more options" section.
> hope it helps.

Hey, thank you for your input. Unfortunately display window doesn't even start.

**Comment by pumpaci** [comment]
> Windows 8 (64 bit), Processing 3.5.4, Python Mode 3056.
> Display window doesn't show up, not even with simple test scripts. Everything works alright in java mode though. I tried to block wifi connection/firewall and reinstall processing with older stable versions.

Hey, can you find a solution.?

**Comment by pumpaci** [comment]
> Windows 8 (64 bit), Processing 3.5.4, Python Mode 3056.
> Display window doesn't show up, not even with simple test scripts. Everything works alright in java mode though. I tried to block wifi connection/firewall and reinstall processing with older stable versions.

Hey, did you find a solution.?

**Comment by wutiship** [comment]
> 
> 
> > Windows 8 (64 bit), Processing 3.5.4, Python Mode 3056.
> > Display window doesn't show up, not even with simple test scripts. Everything works alright in java mode though. I tried to block wifi connection/firewall and reinstall processing with older stable versions.
> 
> Hey, did you find a solution.?

Hey, i have not unfortunately..

**Comment by villares** [comment]
Have you checked if your usernames or directory paths have non-ascii characters?
Tried logging in as guest on Windows?

**Comment by wutiship** [comment]
> 
> 
> Have you checked if your usernames or directory paths have non-ascii characters?
> Tried logging in as guest on Windows?

Yes both using only ascii characters and as a guest couldnt get a result either. I have also updated it to 3059. Thanks for your input

**Comment by JosToneu** [comment]
Hey! I've found a solution to the problem! At least it worked for me. 

My story (with dramatism):
I had a program that was running out of memory heap space so I decided to INCREASE MAXIMUM AVAILABLE MEMORY to 1500Mb (You can modify it in File: Preferences)  . It didn't work anyway so I closed processing and went crying in the corner of my bedroom.
After a couple of days, I woke up fresh like a rose and while trying to run it again (just in case) not even the errors showed up and neither did the display window. After closing the antivirus, reinstalling everything and shutting down background apps, there was still no result. At this point, hopeless, I've remembered the available memory thing. So I reset it to the previous value and my beloved error messages reappeared as well as the display window once those errors were fixed. 

Fast answer: 
Check your INCREASE MAXIMUM AVAILABLE MEMORY value and tinker with it until the display window shows up again. 

![image](https://user-images.githubusercontent.com/66063120/83125741-32918880-a0d8-11ea-85f7-a32291c8d505.png)

Thanks for reading.
